### PR TITLE
PLATFORM-4289 wiki descriptions

### PR DESCRIPTION
**Pull request by harnash**

### CODE REVIEW

File: extensions/wikia/WikiDescription/WikiDescriptionService.class.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace WikiDescription;
+
+class WikiDescriptionService {
+	const WIKI_DESCRIPTION_SITE_META = 'wiki-description-site-meta';
+
+	public static function getDescription( string $site_name ): string {
```
Comment (Grunny) [inline extensions/wikia/WikiDescription/WikiDescriptionService.class.php]:
```suggestion
	public static function getDescription( string $siteName ): string {
```

File: extensions/wikia/WikiDescription/WikiDescriptionService.class.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+		return wfMessage( self::WIKI_DESCRIPTION_SITE_META, [ $site_name ] );
```
Comment (Grunny) [inline extensions/wikia/WikiDescription/WikiDescriptionService.class.php]:
```suggestion
		return wfMessage( self::WIKI_DESCRIPTION_SITE_META, [ $siteName ] );
```

File: includes/wikia/Extensions.php [hunk 3]
```diff
@@ -1810,3 +1810,5 @@
 if ( !empty ( $wgEnableHydralyticsExt ) ) {
 	include "$IP/extensions/wikia/Hydralytics/Hydralytics.setup.php";
 }
+
+include_once ( "${IP}/extensions/wikia/WikiDescription/WikiDescription.setup.php" );
```
Comment (Grunny) [inline includes/wikia/Extensions.php]:
```suggestion
include_once "$IP/extensions/wikia/WikiDescription/WikiDescription.setup.php";
```

File: extensions/wikia/WikiDescription/WikiDescription.setup.php [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+$dir = dirname(__FILE__) . '/';
```
Comment (Grunny) [inline extensions/wikia/WikiDescription/WikiDescription.setup.php]: `__DIR__` is preferred in modern PHP.

File: extensions/wikia/WikiDescription/WikiDescription.setup.php [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+$app = F::app();
```
Comment (Grunny) [inline extensions/wikia/WikiDescription/WikiDescription.setup.php]: Unused.

**Review by Grunny — CHANGES_REQUESTED** [review]

File: extensions/wikia/WikiDescription/WikiDescription.setup.php (hunk 4 above)
Comment (harnash) [reply extensions/wikia/WikiDescription/WikiDescription.setup.php]: Fix 48882859359

File: extensions/wikia/WikiDescription/WikiDescription.setup.php (hunk 5 above)
Comment (harnash) [reply extensions/wikia/WikiDescription/WikiDescription.setup.php]: Fix 48882859359

File: extensions/wikia/WikiDescription/WikiDescription.setup.php [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<?php
+
+$wgAutoloadClasses['WikiDescription\WikiDescriptionService'] =  __DIR__ . 'WikiDescriptionService.class.php';
```
Comment (Grunny) [inline extensions/wikia/WikiDescription/WikiDescription.setup.php]:
```suggestion
$wgAutoloadClasses['WikiDescription\WikiDescriptionService'] =  __DIR__ . '/WikiDescriptionService.class.php';
```

File: extensions/wikia/WikiDescription/WikiDescription.setup.php [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+
+$wgExtensionMessagesFiles['WikiDescriptionService'] = __DIR__ . 'WikiDescription.i18n.php';
```
Comment (Grunny) [inline extensions/wikia/WikiDescription/WikiDescription.setup.php]:
```suggestion
$wgExtensionMessagesFiles['WikiDescriptionService'] = __DIR__ . '/WikiDescription.i18n.php';
```

**Review by Grunny — CHANGES_REQUESTED** [review]

**Review by Grunny — APPROVED** [review]

**Pull request merged**